delivery_schenker/models/delivery_carrier.py

```diff
@@ -266,7 +266,7 @@ def _prepare_schenker_address(
         """
         vals = {
             "type": address_type,
-            "name1": partner.name,
+            "name1": partner.name or partner.commercial_partner_id.name,
```

Comment:
```suggestion
            "name1": partner.name or partner.parent_name
```

Thanks for your contribution.
But the commercial partner name should not be used, because this belongs maybe not to the actual address.
You should better use partner.parent_name. Like it is done in the contact widget of odoo. https://github.com/odoo/odoo/blob/cc0060e889603eb2e47fa44a8a22a70d7d784185/odoo/addons/base/views/ir_qweb_widget_templates.xml#L8

Reply:
cc @jbaudoux